### DISCUSSION TITLE
关于OSS的元数据同步

### DISCUSSION THREAD

**Discussion by jack161641** [juicedata/juicefs#3127] [Q&A] [answered]
我使用OSS作为后端存储的话，非juice client写入到OSS的数据，也就是通过接口也好，其他方式也好，直接存入OSS的数据，怎么同步到juice client里，让juice client可访问？相信这种场景应该很多啊

**Comment by zhijian-pro** [comment, marked as answer]
暂时不支持读取对象存储中已有的数据
